data/human/human missions.txt

```diff
@@ -107,7 +107,269 @@ mission "Pookie, Part 2"
+			label rememberdecline
+			``
+				decline
+	to complete
+		never
+
+
+
+mission "Pookie, Part 3a"
+	invisible
+	source "Freedom"
+	destination "Hestia"
+	to offer
+		has "Pookie, Part 2: done"
+	on complete
+		fail "Pookie, Part 3"
+
+
+
+mission "Pookie, Part 4"
+	repeat
+	landing
+	source
+		attributes "paradise"
+		not planet "Hestia"
+	to offer
+		has "Pookie, Part 3: active"
+	on offer
+		conversation
+			branch repeat
+				has "Pookie, Part 4: offered"
+			`After landing on <origin>, you look for a way to reach as many people on the planet as possible in order to find Puny's owner. One of the receptionists at a planetary information desk says that she can put out a notice for a lost dog on a planetwide notice board, if you're able to put in a detailed enough description of Puny. You describe the poodle's appearance to the best of your abilities, and also mention how she was traveling for a vacation when she was abducted.`
+			`	When you check back with the receptionist the next morning, she tells you that nobody has approached the offer.`
+				decline
+			label repeat
+			`You put a description of Puny on a planetwide notice board. By the time the next day arrives, you've received no offers in response.`
+				decline
+
+
+
+mission "Pookie, Part 5"
+	landing
+	source "Hestia"
+	to offer
+		has "Pookie, Part 3a: done"
+	on offer
+		conversation
+			branch repeat
+				has "Pookie, Part 4: offered"
+			`When you land on <origin>, you look for a way to reach as many people on the planet as possible in order to find Puny's owner. One of the receptionists at a planetary information desk says that she can put out a notice for a lost dog on a planetwide notice board, if you're able to put in a detailed enough description of Puny. You describe the poodle's appearance to the best of your abilities, and also mention how she was meant to be transported to a vacation location before being abducted.`
+				goto next
+			label repeat
+			`You approach a receptionist to put a description of Puny on a planetwide notice board.`
+			label next
+			`	The next day, you visit the receptionist. When she sees you, she tells you that one message came in response, saying that the dog was theirs. You copy the address provided and hail down a taxi.`
+			`	The address brings you to a stark white mansion upon a cliff overlooking the ocean. A path snakes down from the entrance of the mansion to a large, gold-plated gate. You bring Puny out from the taxi and ring the doorbell next to the gate. A minute later, a servant in a prim suit approaches.`
+			choice
+				`	"I'm <first>, and I'm here to return a missing pet."`
+				`	(Wait quietly.)`
+					goto inside	
+			`	The servant says, "Ah, you're the one with the dog. Come inside, mistress has been expecting you."`
+				goto enter
+			label inside
+			`	When he spots you, the servant says, "Welcome. Come inside, mistress has been expecting you."`
```

Comment:
I would expect Puny to recognize the house and to recognize Walters (and respond happily when recognizing Walters), I would even expect Puny to recognize Pemefrey (even when Pemefrey doesn't recognize Puny).